### DISCUSSION TITLE
How to achieve element replacement through "dpgen simplify"

### DISCUSSION THREAD

**Discussion by h757062840** [deepmodeling/dpgen#1482] [Q&A]
I  have a Cu-C-O dataset and would like to replace the Cu element with Au using dpgen simplify. What should I do? Directly replaces the pseudopotentias in FP step? Or make adjustments to "type_map"? 
The example of the  tutorial is only used for fine-tuning input parameters and does not involve thie replacement of element.

**Comment by njzjz** [comment]
I don't think `dpgen simplify` is designed for this purpose.
